install/Vagrantfile

```diff
@@ -74,13 +61,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provision :shell, :path => "./scripts/solr.sh", :args => home_dir
   config.vm.provision :shell, :path => "./scripts/loris.sh", :args => home_dir
   config.vm.provision :shell, :path => "./scripts/composer.sh", :args => home_dir
+  config.vm.provision :shell, :path => "./scripts/drupal-console.sh", :args => home_dir
   config.vm.provision :shell, :path => "./scripts/drupal.sh", :args => home_dir
   config.vm.provision :shell, :path => "./scripts/fcrepo.sh", :args => home_dir
   config.vm.provision :shell, :path => "./scripts/blazegraph.sh", :args => home_dir
   config.vm.provision :shell, :path => "./scripts/alpaca.sh", :args => home_dir
   config.vm.provision :shell, :path => "./scripts/karaf.sh", :args => home_dir
-  config.vm.provision :shell, :path => "./scripts/islandora-karaf-components.sh", :args => home_dir
-  config.vm.provision :shell, :path => "./scripts/config.sh", :args => home_dir
```

Comment:
And actually, without the islandora karaf modules, nothing pulls in fcrepo-indexing-triplestore service. So we probably need to `repo:add` and `feature:install` that particular service to support the configuration parameters we are changing in `config.sh`